bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTHH_5f_LO/TTHH_5f_LO_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm-no_b_mass
+define p = g u c d s u~ c~ d~ s~
```

Comment:
please remove proton and jet definitions, and follow these instructions https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T